schema/ATSpectrograph.yaml

```diff
@@ -29,26 +29,87 @@ properties:
     decription: Tolerance for positioning the linear stage.
     type: number
     default: 0.01
+
   filters:
-    decription: Currently installed filter names.
-    type: array
-    uniqueItems: true
-    items:
-      type: string
-    default:
-     - empty_1
-     - empty_2
-     - diffuser
-     - empty_3
+    type: object
+    properties:
+      filter_name:
+        decription: Currently installed filter names.
+        default:
+          - empty_1
+          - empty_2
+          - diffuser
+          - empty_3
+        type: array
+        uniqueItems: true
+        items:
+          type: string
+        minItems: 4
+        maxItems: 4
+
+      filter_central_wavelength:
+        description: >
+          Wavelength for which optical system will be opimized in units of nm. Approximations are sufficient as the
+          focus dependence on wavelength is weak.
+        type: array
+        items:
+          type: number
+        minItems: 4
+        maxItems: 4
+        default:
+          - 700
+          - 701
+          - 702
+          - 703
+
+      filter_focus_offset:
+        description: >
+          Focus offset to be applied on the secondary mirror in units of um, relative to no glass being installed.
+          Positive values push the secondary down and increase the back focal distance, therefore adding glass
+          thickness will result in positive focus offsets.
+        type: array
+        uniqueItems: false
+        items:
+          type: number
+        minItems: 4
+        maxItems: 4
+        default:
+          - 0.0
+          - 1.0
+          - 2.0
+          - 3.0
+
   gratings:
-    decription: Currently installed grating names.
-    type: array
-    uniqueItems: true
-    items:
-      type: string
-    default:
-      - empty
-      - ronchi170lpmm
-      - ronchi90lpmm
-      - holo_etu1
+    type: object
+    properties:
+      grating_name:
+        decription: Currently installed grating names.
+        default:
+          - empty
+          - ronchi170lpmm
+          - ronchi90lpmm
+          - holo_etu1
+        type: array
+        uniqueItems: true
+        items:
+          type: string
+        minItems: 4
+        maxItems: 4
+      grating_focus_offset:
+        description: >
+          Focus offset to be applied on the secondary mirror in units of um, relative to no glass being installed.
+          Positive values push the secondary down and increase the back focal distance, therefore adding glass
+          thickness will result in positive focus offsets.
+        type: array
+        uniqueItems: false
+        items:
+          type: number
+        minItems: 4
+        maxItems: 4
+        default:
```

Comment:
I think the default should be 0. for all as well...

Reply:
see comment above. These defaults aren't associated with a real config, so why do they need to be 100% physically accurate?